config/externalname.go

```diff
@@ -486,10 +486,11 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	"aws_iam_access_key":       config.IdentifierFromProvider,
 	"aws_iam_instance_profile": config.NameAsIdentifier,
 	// arn:aws:iam::123456789012:policy/UsersManageOwnCredentials
-	"aws_iam_policy": iamPolicy(),
-	"aws_iam_user":   config.NameAsIdentifier,
-	"aws_iam_group":  config.NameAsIdentifier,
-	"aws_iam_role":   config.NameAsIdentifier,
+	"aws_iam_policy":      iamPolicy(),
+	"aws_iam_user":        config.NameAsIdentifier,
+	"aws_iam_group":       config.NameAsIdentifier,
+	"aws_iam_role":        config.NameAsIdentifier,
+	"aws_iam_role_policy": config.IdentifierFromProvider,
```

Comment:
In registry [documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role_policy#import), it seems that the id format is: `role_name:role_policy_name`.

Do you think we can construct it by using the `name` and `role` fields? Maybe you have already tried this, but I wanted to ask you anyway.


```suggestion
	"aws_iam_role_policy": config.TemplatedStringAsIdentifier("name", "{{ .parameters.role }}:{{ .external_name }}"),
```